driver-core/src/test/java/com/datastax/driver/core/MockClocks.java

```diff
@@ -21,7 +21,7 @@
         int calls;
 
         @Override
-        public long currentTime() {
+        public long currentTimeMicros() {
```

Comment:
We could probably move the 'public' modifiers from these since the parent class is not public right?  (we'd probably need to add from clirr exceptions though, doesn't seem important).